core/tests/test_utils.py

```diff
@@ -466,6 +466,26 @@ def get_updated_param_dict(
                 obj_type, new_param_dict)
         return new_param_dict
 
+    def get_cache_slug_filepath(self):
```

Comment:
Ah OK. Can we call it get_static_asset_filepath instead? Cache slug seems
like an implementation detail (to the caller).

On Saturday, July 16, 2016, Vishal Gupta <notifications@github.com> wrote:

> In core/tests/test_utils.py
> <https://github.com/oppia/oppia/pull/2208#discussion_r71070088>:
>
> > @@ -466,6 +466,26 @@ def get_updated_param_dict(
> >                  obj_type, new_param_dict)
> >          return new_param_dict
> >
> > +    def get_cache_slug_filepath(self):
>
> get_static_asset_url uses forward slashes since those are paths that are
> used by the browser, whereas get_cache_slug_filepath returns the path to
> file on a disk, using forward / backward slashes depending on the OS.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/oppia/oppia/pull/2208/files/aa4be4efc52f62354013d697197d9835201afd39#r71070088>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKFeyoxi8uF0FaWEv6j1pF_URHR1AscXks5qWSYNgaJpZM4JDv96>
> .
>